MTA-6889 CLI refactored commands (#372)
migtools/mta-documentation: docs/topics/mta-cli/ref_backward-compatibility.adoc

// Module included in the following assemblies:
//
// * assemblies/cli-guide/assembly_about-the-command-line-tool.adoc

:_mod-docs-content-type: REFERENCE
[id="backward-compatibility_{context}"]
= Compatibility with deprecated commands

[role="_abstract"]

> To ensure that the restructured commands are compatible with the previous command structure, the {ProductFullName} CLI converts an unsupported command option to the new command.

Reviewer: ```suggestion
To ensure compatibility with earlier versions, the {ProductFullName} command-line interface (CLI) converts unsupported command options to the equivalent supported commands.
```

For example, if you enter:

> [subs="+quotes"]

Reviewer: ```suggestion
```

Author: Hey Andy,
Thanks but I'm a little uncertain because I can still see [subs="+quotes"] in the command block examples in the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#commands-in-code-blocks). WDYT?

----

> *$ mta-cli analyze --list-targets*

Reviewer: ```suggestion
$ mta-cli analyze --list-targets
```

The Red Hat SSG was recently updated to remove the requirement to enclose commands in asterisks for bold formatting within code blocks, as this practice can break commands that contain wildcard characters. Therefore, you should remove the asterisks and the [subs="+quotes"] attribute entirely.

----

The {ProductShortName} CLI processes the command as:

> [subs="+quotes"]

Reviewer: ```suggestion
```

Author: Same as above, please clarify if I missed something.

----

> *$ mta-cli rules list-targets*

Reviewer: ```suggestion
$ mta-cli rules list-targets
```
